### PR TITLE
Add aaron to build/OWNERS as approvers

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

**What type of PR is this?**
Adding more reviewers/approvers for build/

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: build/OWNERS
Comment (cblecker) [inline build/OWNERS:11]: This is a dupe

**Review by cblecker — COMMENTED** [review]
/hold

Y'all are trusted contribs, but would love explicit "I am okay with this" from all involved. `build/` has **lots** of load-bearing dragons including the root-level Makefiles. Approving for this dir should be done with extra care.

**Review by justaugustus — CHANGES_REQUESTED** [review]
@dims -- I'm already an approver here, so I'd:

- drop the additional me's from the PR
- remove yourself from approvers

**Comment by spiffxp** [comment]
lol guess what I just opened https://github.com/kubernetes/kubernetes/pull/93935

**Comment by dims** [comment]
LOL @spiffxp let me close this one

**Pull request closed without merging**
